src/test/java/iris/DeadlineTest.java

```diff
@@ -0,0 +1,46 @@
+package iris;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+import iris.task.Deadline;
+
+public class DeadlineTest {
```

Comment:
General question (and may have missed it also), but will you be doing a test on storage? Like if it successfully read/write to storage file?